python/sbp/file_io.py

```diff
@@ -47,28 +47,33 @@ class MsgFileioReadRequest(SBP):
 MSG_FILEIO_READ_RESPONSE message where the message length field
 indicates how many bytes were succesfully read. If the message is
 invalid, a followup MSG_PRINT message will print "Invalid fileio
-read message".
+read message". The sequence number in the request will be returned
+in the response.
 
 
   Parameters
   ----------
   sbp : SBP
     SBP parent object to inherit from.
+  sequence : int
+    Read sequence number
   offset : int
     File offset
   chunk_size : int
     Chunk size to read
   filename : string
-    Name of the file to read from (NULL padded)
+    Name of the file to read from
   sender : int
     Optional sender ID, defaults to SENDER_ID (see sbp/msg.py).
 
   """
   _parser = Struct("MsgFileioReadRequest",
+                   ULInt32('sequence'),
                    ULInt32('offset'),
                    ULInt8('chunk_size'),
-                   String('filename', 20),)
+                   greedy_string('filename'),)
```

Comment:
Does this get changed to `GreedyString` when your Construct PR makes it into upstream?